src/instrumentations/extentions/redis.ts

```diff
@@ -66,21 +66,25 @@ function createRedisResponseHook(
     span: Span,
     cmdName: RedisCommand['command'],
     cmdArgs: RedisCommand['args'],
-    responseInfo
+    responseInfo: unknown
   ) => {
     const spanContext = span.spanContext();
     if (!isSpanContextValid(spanContext)) {
       return;
     }
     switch (cmdName) {
-      case 'hkeys': {
-        span.setAttribute('db.command_arguments', JSON.stringify(cmdArgs));
-        span.setAttribute(
-          'db.command.response',
-          JSON.stringify(responseInfo as Array<String>)
-        );
+      case 'hkeys':
+      case 'hset':
+      case 'hget':
+      case 'del':
+      case 'hgetall':
+      case 'hincrby':
+      case 'exec':
+        span.setAttribute('db.command.arguments', JSON.stringify(cmdArgs));
+        span.setAttribute('db.command.response', JSON.stringify(responseInfo));
         break;
-      }
+      case 'multi':
+        span.setAttribute('db.command.response', responseInfo as string);
```

Comment:
I think the default behavior should be parse arguments and responses as JSON
And check only for 'multi' case, So if we were missed some commands or new commands were developed we will automatically support them